pallets/gear/src/benchmarking/tests/lazy_pages.rs

```diff
@@ -221,7 +221,7 @@ where
                 // Generate load
                 let addr = rng.gen_range(0..max_addr - access_size) as i32;
                 instrs.push(Instruction::I32Const(addr));
-                instrs.push(Instruction::I32Load(2, 0));
+                instrs.push(Instruction::I32Load(MemArg::i32()));
```

Comment:
maybe `OffsetArg::i32()`?

Reply:
Name is not suitable because it's both align and offset (and memory index in the future)